repo/packages/C/cassandra/1/command.json

```diff
@@ -0,0 +1,6 @@
+{
+  "pip": [
+    "dcos<1.0",
```

Comment:
I changed this to dcos==0.1.13 which is the version specified in dcos-cassandra cli. This fixes @kensipe's issue.